libp2p/protocols/ping.nim

```diff
@@ -0,0 +1,95 @@
+## Nim-LibP2P
+## Copyright (c) 2021 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+import options, random
```

Comment:
we avoid the `std/random` module and use bearssl instead (look for `rng`)

also, all std lib imports should be imported with an explicit `std` prefix - `std/[options, ...]`

Reply:
Seems a bit wasteful to use a crypto rand generator for a simple ping?
And std/random is already used in the libp2p, wouldn't have used it otherwise ;)

But sure, can switch to bearssl

Reply:
avoiding `std/random` makes the code easier to audit in general, at no real loss - ie there's nothing really wasteful about it except a few CPU cycles which - `std/random` usage is slowly getting replaced with the crypto rng throughout, also to avoid accidental uses in places where it matters more directly - the question is better asked in the other direction: what special reasons are there to use an insecure rng here? If there are, they should be documented.